patients/repo.go

```diff
@@ -334,6 +334,35 @@ func (r *repository) UpdateEmail(ctx context.Context, userId string, email *stri
 	return err
 }
 
+func (r *repository) UpdateLastReviewed(ctx context.Context, patientUpdate PatientUpdate) (*Patient, error) {
+	clinicObjId, _ := primitive.ObjectIDFromHex(patientUpdate.ClinicId)
+	opts := options.FindOneAndUpdate().SetReturnDocument(1) // return document AFTER edit
+	selector := bson.M{
+		"clinicId": clinicObjId,
+		"userId":   patientUpdate.UserId,
+	}
+
+	patient := patientUpdate.Patient
+	patient.UpdatedTime = time.Now()
+
+	update := bson.M{
```

Comment:
This will reset fields without `omitempty` to their defaults (e.g. `requireUniqueMrn`).

Reply:
moved away from patientUpdate